src/Core/Data/RecipeReview.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace mbsoft.BrewClub.Data
+{
+	public class RecipeReview
+	{
+
+		public UserProfile Author { get; set; }
+
+		public string Comments { get; set; }
+		
+		public bool AuthorBrewedIt { get; set; }
+			
+
```

Comment:
Parent IDs aren't required, EF handles the relationship automatically when you use a class within another class.